sdk/cosmosdb/cosmos/test/functional/container.spec.ts

```diff
@@ -451,3 +451,51 @@ describe("createIfNotExists", function() {
     assert.equal(def.id, readDef.id);
   });
 });
+
+describe("container.readOffer", function() {
+  let containerWithOffer: Container;
+  let containerWithoutOffer: Container;
+  let container2WithOffer: Container;
+  let container2WithoutOffer: Container;
+  const containerRequestWithOffer: ContainerRequest = {
+    id: "sample",
+    throughput: 400
+  };
+  const containerRequest: ContainerRequest = {
+    id: "sample-offerless"
+  };
+  let offerDatabase: Database;
+  before(async function() {
+    offerDatabase = await getTestDatabase("has offer", undefined);
+    containerWithOffer = await getTestContainer(
+      "offerContainer",
+      undefined,
+      containerRequestWithOffer
+    );
+    containerWithoutOffer = await getTestContainer("container", undefined, containerRequest);
+    const response1 = await offerDatabase.containers.create(containerRequest);
+    const response2 = await offerDatabase.containers.create(containerRequestWithOffer);
+    container2WithOffer = response1.container;
+    container2WithoutOffer = response2.container;
+  });
+  describe("database does not have offer", function() {
+    it("has offer", async function() {
+      const offer: any = await containerWithOffer.readOffer();
```

Comment:
Not happy about these `offer: any` anys. I believe this is another relic of the imports pointing to the built files rather than the definition. I'm unable to get this to build locally without serious commenting, it may even pass in CI without these anys. Will test that

Reply:
nb: importing from the built files instead of src is the issue, but deferring for the next PR as a standalone fix